src/install/config/python.yaml

```diff
@@ -0,0 +1,183 @@
+agentName: python
+agentType: APM
+title: 'Monitor your Python application'
+metaDescription: 'Here are several ways to install the Python APM agent'
+introFilePath: 'src/install/python/python-agent-intro.mdx'
+appInfo:
+  - optionType: python-options
+    label: 'Choose the option that best fits your environment'
+    placeholder: 'Please select one:'
+    options:
+      - value: 'unsupported-framework'
+        displayName: 'Unsupported WSGI framework'
+      - value: 'non-web-app'
+        displayName: 'Non-web app'
+      - value: 'ajp'
+        displayName: 'AJP'
+      - value: 'aws'
+        displayName: 'AWS Lambda'
+      - value: 'cherrypy'
+        displayName: 'CherryPy'
+      - value: 'daphne'
+        displayName: 'Daphne'
+      - value: 'fastapi'
+        displayName: 'FastAPI'
+      - value: 'fastcgi'
+        displayName: 'FastCGI'
+      - value: 'gunicorn'
+        displayName: 'Gunicorn'
+      - value: 'hypercorn'
+        displayName: 'Hypercorn'
+      - value: 'mod-wsgi'
+        displayName: 'mod_wsgi'
+      - value: 'mod-wsgi-express'
+        displayName: 'mod_wsgi-express'
+      - value: 'paste'
+        displayName: 'Paste'
+      - value: 'scgi'
+        displayName: 'SCGI'
+      - value: 'starlette'
+        displayName: 'Starlette'
+      - value: 'tornado'
+        displayName: 'Tornado'
+      - value: 'uvicorn'
+        displayName: 'Uvicorn'
+      - value: 'uvicorn-docker'
+        displayName: 'Uvicorn with Docker'
```

Comment:
I found it a little odd at first glance to see `Uvicorn with Docker` and not just `Docker`. I understand what we are going for here though (Uvicorn is just the example server that is being integrated) I wonder if this might be confusing to users as well and if you had any thoughts on how to mitigate that confusion?